Automattic/jetpack
# Social: Add media validation for Nextdoor

This small change updates our restrictions file, so it contains the the constraints for the nextdoor connection.

### Other information:


## Jetpack product discussion


https://github.com/orgs/Automattic/projects/733/views/2?pane=issue&itemId=41283346
## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* Sanity check the code
* Compare the added values with the [documentation](https://documenter.getpostman.com/view/11147684/2s93K1qKws#dc813de1-0238-4cdf-8708-ff7bcd3c8988)
